navigator/backend/src/main/scala/com/digitalasset/navigator/model/Model.scala

```diff
@@ -194,15 +195,23 @@ final case class Template(
     id: DamlLfIdentifier,
     choices: List[Choice],
     key: Option[DamlLfType],
+    implementedInterfaces: Set[DamlLfIdentifier] = Set.empty,
```

Comment:
I'd go ahead and do this at the callers. Just a soft suggestion, though.

```suggestion
    implementedInterfaces: Set[DamlLfIdentifier],
```